docs/ConfigureScalarDB.md

```diff
@@ -0,0 +1,46 @@
+# Configure Scalar DB Server
+
+This guide explains how to create database schema and configure database properties in helm charts for Scalar DB server deployment.
+
+## Create schema
+
+To create database schema, [schema tool for Scalar DB](https://github.com/scalar-labs/scalardb/tree/master/tools/scalar-schema) can be used to create the database schema.
```

Comment:
I'm not sure. Why do we not use helm charts for schema loading?
https://github.com/orgs/scalar-labs/packages/container/package/scalardb-schema-loader

Reply:
I will check and update regarding this.